tests/components/sonos/test_media_player.py

```diff
@@ -59,3 +60,17 @@ async def test_device_registry(hass, config_entry, config, soco):
     assert reg_device.manufacturer == "Sonos"
     assert reg_device.suggested_area == "Zone A"
     assert reg_device.name == "Zone A"
+
+
+async def test_entity_basic(hass, config_entry, discover):
+    """Test basic state and attributes."""
+    await setup_platform(hass, config_entry, {})
+
+    state = hass.states.get("media_player.zone_a")
+    assert state.state == STATE_IDLE
+    attributes = state.attributes
+    assert attributes["friendly_name"] == "Zone A"
+    assert attributes["is_volume_muted"] is False
+    assert attributes["night_sound"] is True
+    assert attributes["speech_enhance"] is False
```

Comment:
This test fails.

Reply:
Works better when the file is uploaded before commit :)

Fixed